src/pages/404.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
Does gatsby pick this up if it is a jsx file?

Reply:
Seems to work locally, but haven't tested in a build. According to the docs, it should work. https://www.gatsbyjs.org/docs/add-404-page/ 
If you prefer to stick with .js or switch the others to jsx as well, let me know